skt/__init__.py

```diff
@@ -498,6 +498,17 @@ def getrelease(self):
         return krelease
 
     def mktgz(self, clean=True, timeout=60 * 60):
+        """
+        Build kernel and modules, after that, pack everything into a tarball.
+
+        Args:
+            clean:      If it's True, run the `mrproper` target before build.
+            timeout:    Max time in seconds will wait for build, 1h by default.
+        Returns:
+            The full path of the tarball generated.
+        Raises:
+            CommandTimeoutError: When building kernel takes too long.
```

Comment:
I think this needs to say "takes longer than the specified timeout".

Reply:
Done